db/migrate/20201030102020_enlarge_report_status.rb

```diff
@@ -0,0 +1,29 @@
+class EnlargeReportStatus < ActiveRecord::Migration[6.0]
+  def up
+    remove_index :reports, :status
```

Comment:
The index is unused, see the description. We don't need it, so let's save some space in the db.